source/mir/cv/core/simd.d

```diff
@@ -0,0 +1,77 @@
+/+
+    Utilities to help use SIMD vectors.
++/
+
+module mir.cv.core.simd;
```

Comment:
> Front-end DCV users would be able to use DMD because extern(C) interface.

I think you meant to say 'Front-end DCV users would be able to use **mir-cv**...'? If thats the case, I concur, I understood that was the idea from the beginning.

> I would propose another approach that is used in mir-glas, see glas/internal/config.

Nice! It's really cool there are already some modules that we can reuse in mir libs. I'll study this module and see how I can fit it here. Should this also be copied to mir-rt?